src/bo4e/enum/themengebiet.py

```diff
@@ -8,64 +8,64 @@ class Themengebiet(StrEnum):
     Über dieses ENUM kann eine thematische Zuordnung, beispielsweise eines Ansprechpartners, vorgenommen werden.
     """
 
-    ALLGEMEINER_INFORMATIONSAUSTAUSCH = "ALLGEMEINER_INFORMATIONSAUSTAUSCH"  # Allgemeiner Informationsaustausch
-    AN_UND_ABMELDUNG = "AN_UND_ABMELDUNG"  # An- und Abmeldung
-    ANSPRECHPARTNER_ALLGEMEIN = "ANSPRECHPARTNER_ALLGEMEIN"  # Ansprechpartner Allgemein
-    ANSPRECHPARTNER_BDEW_DVGW = "ANSPRECHPARTNER_BDEW_DVGW"  # Ansprechpartner BDEW/DVGW
-    ANSPRECHPARTNER_IT_TECHNIK = "ANSPRECHPARTNER_IT_TECHNIK"  # Ansprechpartner IT/Technik
-    BILANZIERUNG = "BILANZIERUNG"  # Bilanzierung
-    BILANZKREISKOORDINATOR = "BILANZKREISKOORDINATOR"  # Bilanzkreiskoordinator
-    BILANZKREISVERANTWORTLICHER = "BILANZKREISVERANTWORTLICHER"  # Bilanzkreisverantwortlicher
+    ALLGEMEINER_INFORMATIONSAUSTAUSCH = "ALLGEMEINER_INFORMATIONSAUSTAUSCH"  #: Allgemeiner Informationsaustausch
+    AN_UND_ABMELDUNG = "AN_UND_ABMELDUNG"  #: An- und Abmeldung
+    ANSPRECHPARTNER_ALLGEMEIN = "ANSPRECHPARTNER_ALLGEMEIN"  #: Ansprechpartner Allgemein
+    ANSPRECHPARTNER_BDEW_DVGW = "ANSPRECHPARTNER_BDEW_DVGW"  #: Ansprechpartner BDEW/DVGW
+    ANSPRECHPARTNER_IT_TECHNIK = "ANSPRECHPARTNER_IT_TECHNIK"  #: Ansprechpartner IT/Technik
+    BILANZIERUNG = "BILANZIERUNG"  #: Bilanzierung
+    BILANZKREISKOORDINATOR = "BILANZKREISKOORDINATOR"  #: Bilanzkreiskoordinator
+    BILANZKREISVERANTWORTLICHER = "BILANZKREISVERANTWORTLICHER"  #: Bilanzkreisverantwortlicher
     DATENFORMATE_ZERTIFIKATE_VERSCHLUESSELUNGEN = (
         "DATENFORMATE_ZERTIFIKATE_VERSCHLUESSELUNGEN"  # Datenformate, Zertifikate, Verschlüsselungen
```

Comment:
The `:` is here missing, isn't it?

Reply:
probably my regex search and replace failed for those entries... maybe because they span multiple lines.

Reply:
https://github.com/Hochfrequenz/BO4E-python/pull/81/commits/143447900d39e11946b4587c2c513402727548c5